props/saber_fett263_buttons.h

```diff
@@ -973,34 +892,36 @@ int IntEdit::int_edit_ = 0;
 #endif
 
 // Color List
-static constexpr Color16 color_list_[] = {
-  Red::color(),
-  OrangeRed::color(),
-  DarkOrange::color(),
-  Orange::color(),
-  { 46260, 33410, 0 }, // Gold
-  Yellow::color(),
-  GreenYellow::color(),
-  Green::color(),
-  Aquamarine::color(),
-  Cyan::color(),
-  DeepSkyBlue::color(),
-  DodgerBlue::color(),
-  Blue::color(),
-  { 7710, 15420, 51400 }, // IceBlue
-  { 11102, 92, 53864 }, // Indigo
-  { 24000, 0, 50536 }, // Purple
-  { 30324, 0, 49768 }, // DeepPurple
-  Magenta::color(),
-  DeepPink::color(),
-  { 25700, 25700, 38550 }, // Silver
-  { 21845, 21845, 51400 }, // Glacier
-  { 46260, 46260, 65535 }, // IceWhite
-  LightCyan::color(),
-  Moccasin::color(),
-  LemonChiffon::color(),
-  NavajoWhite::color(),
-  White::color()
+struct ColorListEntry { Color16 color; ColorNumber color_number; };
+
+static constexpr ColorListEntry color_list_[] = {
+  { Red::color(), COLOR_RED },
+  { OrangeRed::color(), COLOR_ORANGERED },
+  { DarkOrange::color(), COLOR_DARKORANGE },
+  { Orange::color(), COLOR_ORANGE },
+  { { 46260, 33410, 0 }, COLOR_GOLD }, // Gold
+  { Yellow::color(), COLOR_YELLOW },
+  { GreenYellow::color(), COLOR_GREENYELLOW },
+  { Green::color(), COLOR_GREEN },
+  { Aquamarine::color(), COLOR_AQUAMARINE },
+  { Cyan::color(), COLOR_CYAN },
+  { DeepSkyBlue::color(), COLOR_DEEPSKYBLUE },
+  { DodgerBlue::color(), COLOR_DODGERBLUE },
+  { Blue::color(), COLOR_BLUE },
+  { { 7710, 15420, 51400 }, COLOR_ICEBLUE }, // IceBlue
```

Comment:
The comments aren't actually helpful anymore since the COLOR_ICEBLUE actually says what the color is.

Reply:
removed